ortho_seq_code/logo_plot.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import pandas as pd
+import logomaker as lm
+import matplotlib.pyplot as plt
+import ortho_seq_code.utils as utils
+import itertools
+import click
+from ortho_seq_code.logo_colors import all_dna, all_rna, all_prot
+
+
+def logo_plot(
+    filename,
+    molecule,
+    out_dir,
+):
+
+    """Program to generate a logo plot of sequence data"""
+    # ---- Import the sequence data. ----
+    (dm, sites, pop_size, seq, alphabets, custom_aa, exc) = utils.get_seq_info(
```

Comment:
You should be able to just use `_, _` for the last two arguments to indicate that they aren't used (unless I missed where they are used which is very possible.